templates/commands/goldentest/verify.go

```diff
@@ -86,7 +102,7 @@ func (c *VerifyCommand) Run(ctx context.Context, args []string) (rErr error) {
 	defer tempTracker.DeferMaybeRemoveAll(ctx, &rErr)
 
 	// Create a temporary directory to render golden tests
-	tempDir, err := renderTestCases(ctx, testCases, c.flags.Location)
+	tempDir, err := renderTestCases(ctx, testCases, templateLocation)
```

Comment:
Same here, we need some way for the user to identify which tests failed to run. The error message doesn't say:

```
failed to render test cases: failed to render golden tests: error reading template spec file: error parsing YAML file spec.yaml: at line 26 column 7: unknown field name "foo"; valid choices are [as from on_conflict paths skip]
exit status 1
```

In general, I think it would be a good idea to pretend that you're a user and execute the CLI for the common failure cases.